docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "Bruno-Testing",
+  "type": "collection"
```

Comment:
It would be beneficial to include more metadata in the 'bruno.json' file, such as a description of the collection, the author, etc. This will make it easier for others to understand the purpose and context of these tests. [medium]